geonode/base/models.py

```diff
@@ -354,6 +354,9 @@ class ResourceBase(PolymorphicModel, PermissionLevelMixin):
     detail_url = models.CharField(max_length=255, null=True, blank=True)
     rating = models.IntegerField(default=0, null=True, blank=True)
 
+    # check
+    can_view_resource = False
```

Comment:
This was created to init the can_view_resource. Yet, I think we could delete it because I'm not sure it is really used...